### DISCUSSION TITLE
Livewire Volt Functional API auto completion

### DISCUSSION THREAD

**Discussion by nerisonpitogo** [microsoft/vscode-discussions#1266] [Extension Development QnA] [answered]
Is there an extension that supports livewire volt functional API auto-completion? Seems like right now, only phpstorm has it.

**Comment by alefragnani** [comment, marked as answer]
Hi @nerisonpitogo ,

Based on a quick search in the extension’s marketplace (https://marketplace.visualstudio.com/search?term=livewire&target=VSCode&category=All%20categories&sortBy=Relevance), it seems none of the Livewire extension does support Volt. I would suggest you to look/ask in any of these extension’s repo for plans on Volt support.

By the way, for future feature requests about VS Code usage, please use the VS Code repository instead (https://github.com/microsoft/vscode/issues/new/choose). This Discussions repo is focused for extension development for now.

Hope this helps

**Reply by burkeholland** [reply]
Marking this as answered for now. @nerisonpitogo if you feel this is a mistake, please feel free to re-open.
